benchmark/lib/grpc/core/stats.pb.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Grpc.Core.Bucket do
```

Comment:
Are these changes intentional?

Reply:
I believe this was already like this, I don't know when or why these files were originally created.

Reply:
I mean the downgrade in the version. It seems that when you re-generated you used 14.0 instead of 14.1